heartbeat/galera

```diff
@@ -605,6 +656,11 @@ galera_promote()
         bootstrap=$(is_bootstrap)
 
         if ocf_is_true $bootstrap; then
+            # The best node for bootstrapping wasn't cleanly shutdown. Allow
+            # bootstrapping anyways
```

Comment:
What are the consequences of this? If it wasn't clean, why are we using it?
I presume if we get to this point it means no nodes are clean. However does that mean we need support intervention?

Reply:
> What are the consequences of this? If it wasn't clean, why are we using it? I presume if we get to this point it means no nodes are clean.

Right

> However does that mean we need support intervention?

Not at this point. We were still able to figure which node has the latest data and we can recover from there. (It's just fallback to the old behaviour of the resource-agent, which is still valid)